upgrade/upgrade-scripts/verify-impl

```diff
@@ -36,6 +37,94 @@ function run_support_info() {
 	return "$rc"
 }
 
+function verify_run_verify_jar() {
+	echo "Verify - Run verify jar."
+
+	/usr/bin/java \
+		-Dlog.dir=/var/delphix/server/upgrade-verify \
+		-Dmdsverify=true \
+		"${VERIFY_DEBUG_OPT}" \
+		-jar /opt/delphix/server/lib/exec/upgrade-verify/upgrade-verify.jar \
+		-d "${opt_o:-${LOG_DIRECTORY}/${CONTAINER}/upgrade_verify.json}" \
+		-f "${opt_f:-1}" \
+		-l "${opt_l:-en-US}" \
+		-v "$IMAGE_VERSION" \
+		-pl 60 -ph 80 \
+		"${VERIFY_LIVE_MDS_OPT}" ||
+		die "'upgrade-verify.jar' failed in verification container"
+
+	#
+	# This name is used by the "upgrade-verify.jar" just executed, so we
+	# cannot change this value without also modifying that JAR.
+	#
+	MDS_SNAPNAME="MDS-CLONE-upgradeverify"
+
+	# TODO : Make sure MDS clone gets cleaned up if verification fails.
+	/opt/delphix/server/bin/dx_manage_pg stop -s "$MDS_SNAPNAME" ||
+		die "failed to stop postgres"
+
+	/opt/delphix/server/bin/dx_manage_pg cleanup -s "$MDS_SNAPNAME" ||
+		die "failed to cleanup postgres"
+}
+
+function verify_stack_startup() {
+	echo "Verify - Run stack startup."
+
+	#
+	# Make sure delphix-mgmt and delphix-postgres aren't running within container.
+	#
+	[[ $(systemctl is-active delphix-mgmt) == 'inactive' ]] ||
+		die "delphix-mgmt service is not inactive."
```

Comment:
Would it be better to capture the result of `systemctl is-active` to a variable, so we can emit the value on failure? e.g.
```
IS_ACTIVE_MGMT=$(systemctl is-active delphix-mgmt)
[[ "$IS_ACTIVE_MGMT" == "inactive" ]] ||
    die "delphix-mgmt service expected to be 'inactive', found to be '$IS_ACTIVE_MGMT'"
```
this might help us better understand a failure more quickly when/if it fails..

same for the postgres check below..